compiler/rustc_feature/src/removed.rs

```diff
@@ -155,9 +155,12 @@ declare_features! (
      Some("removed in favor of `#![feature(marker_trait_attr)]`")),
     (removed, panic_implementation, "1.28.0", Some(44489), None,
      Some("subsumed by `#[panic_handler]`")),
+    /// Allows using `#![plugin(myplugin)]`.
+    (removed, plugin, "1.74.0", Some(29597), None, // njn: version?
```

Comment:
```suggestion
    (removed, plugin, "CURRENT_RUSTC_VERSION", Some(29597), None, // njn: version?
```
?

Reply:
Shouldn't it be `"1.75.0"`?

Reply:
It will be replaced with proper version like in https://github.com/rust-lang/rust/pull/116407
This mechanism was introduced in https://github.com/rust-lang/rust/pull/100591

Reply:
I see, thanks for the explanation.